26_Autoexposure/app_resources/common.hlsl

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2018-2024 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#ifndef _AUTOEXPOSURE_COMMON_INCLUDED_
+#define _AUTOEXPOSURE_COMMON_INCLUDED_
+
+#include "nbl/builtin/hlsl/cpp_compat.hlsl"
+
+struct AutoexposurePushData
```

Comment:
shouldn't a struct for push constants shared between your Luma Meter and Tonemapper supposed to be in the extension headers?